### DISCUSSION TITLE
Use tags with go test command

### DISCUSSION THREAD

**Discussion by gabrielcartellivia** [cucumber/godog#534] [Q&A]
Hi,

I wanna use cucumber tags. How can I do it with `go test` command considering the [deprecation of godog CLI](https://github.com/cucumber/godog/discussions/478)?

E.g.: I wanna execute just the scenario with tag @smoke

My code:
![image](https://user-images.githubusercontent.com/72199278/214673999-eefa78f1-adc1-47f3-9eab-3f12a5b19b0e.png)
![image](https://user-images.githubusercontent.com/72199278/214674066-4e6fdfa5-4751-4c10-90ee-5ff8f4b8c126.png)

**Comment by vearutop** [comment]
Please check https://github.com/cucumber/godog/discussions/495#discussioncomment-3404240.

And then have something like
```
go test ./... -test.v -test.run "TestFeatures/.*"  -godog.tags "@smoke"
```
